### DISCUSSION TITLE
What is your favorite country?

### DISCUSSION THREAD

**Discussion by MK202303** [NAIST-SE/AttractingContributors#1154] [2024 Attraction Contributions]
https://github.com/MK202303/Kanta2

**Comment by Ryota-keda** [comment]
I just made PR.
Could you contribute to my project?
https://github.com/Ryota-keda/Baseball

**Comment by 3207-Rhims** [comment]
https://github.com/3207-Rhims/AttractingContributors-Movie , Would you be kind to make a PR on my project. I made a PR for you. Thank you.

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by r-nishioka** [comment]
I just made PR.
Could you contribute to my project?
[Where would you like to trip in Japan?](https://github.com/r-nishioka/AttractingContributors-Trip.git)

**Comment by Marsall-k** [comment]
I have made a PR on your repo.
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
